test/fixtures/webpack-message-formatting/__snapshots__/index.test.js.snap

```diff
@@ -42,13 +42,13 @@ Object {
 Failed to compile.
 
 ./src/AppCss.css
-Syntax error: Unexpected } (3:2)
+Syntax error: /private/var/folders/qb/k90ylsh13ljdyp0d7_30849w0000gn/T/d6650a73260d68d1eaf908b68d99b855/src/AppCss.css Unexpected } (3:2)
```

Comment:
This is... Not a good change. What happened here?

Reply:
Snapshots represent the result of what the program does. We shouldn't just update the snapshots, we need to understand if they reflect what the program should be doing. If they don't, we need to fix the program.

Why would updating Jest affect webpack snapshots?

Reply:
Ran `../node_modules/.bin/jest -w 2` with `-u` flag to update the snapshots.

Reply:
This is happening to other files too, it looks like the error reporting is different. Like this one

```

Reply:
I understand that you updated the snapshot. The *purpose* of snapshots is to tell us when something changes unexpectedly. Is this change expected? It's not expected to me. So why did this change? This is what I'm asking.

Updating the snapshots without understanding why they updated defeats the whole purpose of snapshots.

Reply:
@gaearon: Right. Snapshots should be updated only when absolutely needed & updating `jest` shouldn't really affect the snapshots 🤔 

I will revert the change but I guess we will have to dig deeper to why some of the CIs instances are failing?

Reply:
We need to figure out whether

- These snapshots are _already failing_ on master
- Or updating Jest caused a difference in at which point we snapshot
- Or updating Jest somehow caused a difference in the output itself

Reply:
As for first, so I just did this:
- Checked out `master`
- Ran `yarn` again
- And ran the snapshots with `../node_modules/.bin/jest -w 2` 

I do get failing snapshots there as well:

```
Snapshot Summary
 › 4 snapshots failed from 1 test suite. Inspect your code changes or re-run jest with `-u` to update them.

Test Suites: 2 failed, 10 passed, 12 total
Tests:       7 failed, 24 passed, 31 total
Snapshots:   4 failed, 8 passed, 12 total
Time:        115.442 s
```

Reply:
OK, then we need to update snapshots on master in a separate PR and also file an issue to investigate why they changed. Then rebase this PR on top.

Reply:
I will revert my change then & only update the timeout.

Reply:
That also works.